### DISCUSSION TITLE
`<picture>`

### DISCUSSION THREAD

**Discussion by LeaVerou** [Devographics/surveys#122] [State of HTML Features]
A responsive images method to control which image resource a user agent presents to a user, based on various factors.

### Code sample

```html
<picture>
    <source srcset="vertical.jpg"
            media="(orientation: portrait)">
    <img src="horizontal.jpg" alt="">
</picture>
```

### MDN URL:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/picture

### Specification URL:

https://html.spec.whatwg.org/multipage/embedded-content.html#the-picture-element

### Any other links?

https://caniuse.com/picture

### How stable is this feature?

Stable (Widely supported for several years)

### Type of feature

HTML element

### What focus area does this feature fall into?

Multimedia

**Comment by nileshprajapati** [comment]
It would be nice to have an option out the image formats commonly used.
